.vuepress/config.js

```diff
@@ -334,37 +381,37 @@ module.exports = {
                   path: "/feature-test/"
                 },
                 {
-                  title: "What is Cosmjs (todo)",
-                  path: "/feature-test/"
+                  title: "What is CosmJS",
+                  path: "/academy/xl-cosmjs/intro.html"
                 },
                 {
-                  title: "Your first CosmJs actions (todo)",
-                  path: "/feature-test/"
+                  title: "Your First CosmJS Actions",
+                  path: "/academy/xl-cosmjs/first-steps.html"
                 },
                 {
-                  title: "Compose complex transactions (todo)",
-                  path: "/feature-test/"
+                  title: "Compose Complex Transactions",
+                  path: "/academy/xl-cosmjs/create-custom.html"
                 },
                 {
-                  title: "Learn to integrate Keplr (todo)",
-                  path: "/feature-test/"
+                  title: "Learn to Integrate Keplr",
+                  path: "/academy/xl-cosmjs/with-kepler.html"
```

Comment:
It will be `with-keplr.html`. And perhaps eventually it should be something other than `xl-cosmjs`, which are really my initials.